test/jit/test_complex.py

```diff
@@ -109,3 +109,30 @@ def fn(x: complex):
 
         for val in complex_vals:
             self.checkScript(fn, (val, ))
+
+    def test_cmath_constants(self):
+        def checkCmathConst(const_name):
+            funcs_template = dedent('''
+                def func():
+                    return cmath.{const}
+                ''')
+
+            funcs_str = funcs_template.format(const=const_name)
+            scope = {}
+            execWrapper(funcs_str, globals(), scope)
+            cu = torch.jit.CompilationUnit(funcs_str)
+            f_script = cu.func
+            f = scope['func']
+
+            res_python = f()
+            res_script = f_script()
+
+            if res_python != res_script:
+                msg = ("Failed on {const_name}. Python: {res_python}, Script: {res_script}"
+                       .format(const_name=const_name, res_python=res_python, res_script=res_script))
+                self.assertEqual(res_python, res_script, msg=msg)
```

Comment:
Looks similar to the testing code from the PR below, consider doing some refactoring.

Reply:
good idea! done